### PR TITLE
VACMS-4262: Remove tome, zippy and va_gov_content_export

### PR DESCRIPTION
**Pull request by indytechcook**
## Description

See #4262. 

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [x] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

---
This change is [Reviewable](https://reviewable.io/reviews/department-of-veterans-affairs/va.gov-cms/4650)

### CODE REVIEW

**Comment by indytechcook** [comment]
Holding until #4620 since there will be conflicts

File: docroot/modules/custom/va_gov_build_trigger/src/WebBuildCommandBuilder.php [hunk 1]
```diff
@@ -185,7 +171,10 @@ protected function getFrontEndResetCommand(string $repo_root) : string {
   }
 
   /**
-   * {@inheritdoc}
+   * Use CMS export.
```
Comment (acrollet) [inline docroot/modules/custom/va_gov_build_trigger/src/WebBuildCommandBuilder.php:174]: Seems like this should be removed as well?

**Review by acrollet — CHANGES_REQUESTED** [review]
Overall looks good, I see a few places where there may be content export related stuff still around:

- an `@see` in `Drupal\va_gov_build_trigger\Command\CommandRunner`
- tome reference in `.github/dependabot.yml`
- references to `useContentExport()` in several files under `Drupal\va_gov_build_trigger`
- CMS export README still referenced in the main README
- (docBlock only) reference in `Drupal\va_gov_build_trigger\SiteStatus\SiteStatusInterface`
- looks like `READMES/devops/deploy-process.md` needs some updates

Nice to see this simplification! 😁

**Comment by indytechcook** [comment]
I'm replacing this PR with https://github.com/department-of-veterans-affairs/va.gov-cms/pull/4716 since there are cross deps between the issues/prs

**Pull request closed without merging**